requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.17.0
+numpy >=1.17.0,<2.0
 protobuf==3.6.1
-pandas==0.24.2
+pandas >=0.24.2,<2.0
```

Comment:
Shouldn't this be `<1.0.0`?

Reply:
honestly, i don't know. i understand not wanting to frivolously update to the latest of upstream libraries for no reason, but 1.0 has been out for over 2 months and i am just trying to get python 3.8 support.